addons/mass_mailing/models/mailing.py

```diff
@@ -74,6 +74,8 @@ def _get_default_mail_server_id(self):
              'Keep it empty if you prefer the first characters of your email content to appear instead.')
     email_from = fields.Char(string='Send From', required=True, store=True, readonly=False, compute='_compute_email_from',
                              default=lambda self: self.env.user.email_formatted)
+    favorite = fields.Boolean('Favorite', copy=False, tracking=True)
+    favorite_date = fields.Datetime('Favorite Date', help='When this mailing was added in the favorites', copy=False)
```

Comment:
Why not a computed (stored editable probably) field ? We could set it when setting favorite, and reset it when removing favorite.

Reply:
with a stored / computed fields, the date will be set when we try to read the value, so it's not that robust imo

e.g. the following test will fail
```python
@users('user_marketing')
def test_mailing_computed_favorite_date(self):
    now = datetime(2020, 1, 2, 18, 0)

    with freeze_time('2020-01-02 18:00'):
        mailing = self.env['mailing.mailing'].create({'name': 'Test', 'subject': 'Test'})

        self.assertFalse(mailing.favorite)
        self.assertFalse(mailing.favorite_date)
        mailing.favorite = True

    with freeze_time('2020-09-09 18:00'):
        mailing.favorite_date

    self.assertEqual(mailing.favorite_date, now)

    print(mailing.favorite_date)
```

An other solution would be to add an inverse on the `favorite` fields, I did it, tell me what you think

Reply:
Don't understand ... we have several fields of that kind, like date_open on lead for example. It is updated when trigger is updated.
```
    @api.depends('user_id')
    def _compute_date_open(self):
        for lead in self:
            lead.date_open = fields.Datetime.now() if lead.user_id else False
```